client/my-sites/upgrades/domain-management/dns/txt-record.jsx

```diff
@@ -55,7 +55,7 @@ const TxtRecord = React.createClass( {
 						name="data"
 						onChange={ this.props.onChange }
 						value={ this.props.fieldValues.data }
-						placeholder={ this.translate( 'e.g. a=b; verification-key=something', { context: 'TXT DNS Record', textOnly: true } ) } />
+						placeholder={ this.translate( 'e.g. %(example)s', { args: { example: 'a=b; verification-key=something' } } ) } />
```

Comment:
I'm not sure whether it's the best option move TXT record out of translation. I know many of them are in English, but still.